CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,76 @@
+# Great Expectations Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, sex characteristics, gender identity and expression,
+level of experience, education, socio-economic status, nationality, personal
+appearance, race, religion, or sexual identity and orientation.
```

Comment:
```suggestion
We as members, contributors, and leaders pledge to make participation in our community a harassment-free
experience for everyone. We value everyone as whole human beings. To foster an atmosphere of kindness,
cooperation, and understanding, we encourage and support participation regardless of age, body size, visible
or invisible disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education,
socio-economic status, nationality, personal appearance, race, religion, sexual identity and orientation, or other
characteristics.
```

Suggestion to make the language slightly more positive--value whole person rather than not discriminate against list (but keep the list)

Reply:
I think this is great!

Reply:
Great--updated!